qiita_pet/templates/sitebase.html

```diff
@@ -241,6 +243,11 @@
                 <li>
                   <a href="http://github.com/biocore/qiita">GitHub</a>
                 </li>
+                <li role="separator" class="divider"></li>
+                <li>
+                  <a href="#"><h6>{{qiita_version}}</h6></a>
```

Comment:
I tried adding in 1 line but it looks horrible. I can try in 2, what do you think?